cGame.h

```diff
@@ -11,11 +11,12 @@
   */
 #pragma once
 
-#include <controls/cMouse.h>
-#include <data/cAllegroDataRepository.h>
-#include <observers/cScenarioObserver.h>
-#include <utils/cRectangle.h>
-#include <utils/cSoundPlayer.h>
+#include "controls/cMouse.h"
+#include "data/cAllegroDataRepository.h"
+#include "definitions.h"
```

Comment:
I feel like this file also might be candidate for removal some time? (ie, we could move things from within `definitions.h` closer to their classes where they are used?). For now 👌 though.

Reply:
Certainly. That's a direction of improvement that's also on my mind.